.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   deploy_staging:
     docker:
-      - image: circleci/node:12.16.1
+      - image: circleci/node:14.9.0
```

Comment:
Do you see anything that can be risky to upgrade 14.x? 

I was thinking we'll stay on the 12.x version of the Node till 14.x becomes LTS.

Reply:
I was thinking the same. I'm not sure really, should I downgrade ?

Reply:
Nodejs says;
>  Production applications should only use Active LTS or Maintenance LTS releases.


Let's stay on the 12.x then. 

14.x will be active LTS on 2020-10-27, we can do it next month

Reply:
Cool I will make the changes